docs/assistant/security-assistant.asciidoc

```diff
@@ -2,4 +2,97 @@
+
+Also, the data you provide to the Security Assistant is _not_ anonymized, and it is stored and processed by the third-party AI provider. This includes any data included in conversations for analysis or context, such as alert or event data, detection rule configurations, queries, and so on. Therefore, be careful about sharing any confidential or sensitive details while using this feature.
+====
+
+.Requirements
+[sidebar]
+--
+* The Elastic Security Assistant is available in {elastic-sec} version 8.8.1 and later.
+
+* This feature requires an https://www.elastic.co/pricing[Enterprise subscription].
+
+* You must have an account with a third-party generative AI provider, which the Security Assistant uses to generate responses. Supported providers are OpenAI (`gpt-3.5-turbo` model) and Azure OpenAI Service (any model).
+--
+
+[discrete]
+[[set-up-ai-assistant]]
+== Set up the Security Assistant
+
+You must complete these steps before you can use the Security Assistant:
+
+. Create an API key with your AI provider to authenticate requests from the Security Assistant. You'll use this in a later step. Refer to the provider's documentation on generating API keys:
++
+* https://platform.openai.com/docs/api-reference[OpenAI]
+* https://learn.microsoft.com/en-us/azure/cognitive-services/openai/reference[Azure OpenAI Service]
+
+. Add the following feature flag to {kib}'s configuration settings:
++
+`xpack.securitySolution.enableExperimental: ['assistantEnabled']` 
++
+The configuration method depends on your deployment type:
++
+* *Self-managed (on-premises) deployments*: Add the feature flag to the `kibana.yml` file, which is used to {kibana-ref}/settings.html[configure {kib}], then restart {kib}.
+* *{ecloud} deployments*: Use the YAML editor in the {ecloud} console to add the feature flag to {cloud}/ec-manage-kibana-settings.html[{kib} user settings].
+
+. Create a {kibana-ref}/gen-ai-action-type.html[Generative AI connector], using the AI provider's API key and URL to configure communication between {elastic-sec} and the provider. You can do this in {kib} at *Stack Management* -> *Connectors*, or from within the Security Assistant.
++
+NOTE: The Generative AI connector type requires the `assistantEnabled` feature flag, and won't be available without it.
+
+[discrete]
+[[start-chatting]]
+== Start chatting
+
+To open the Security Assistant, press *Cmd + ;* (or *Ctrl + ;* in Windows) from anywhere in the {security-app}. This opens the *Welcome* conversation context, where you can ask general questions about {elastic-sec}.
+
+You can also chat with the Security Assistant from several areas of {elastic-sec}, and context-specific data and prompts will populate your conversation.
```

Comment:
Another super nitty edit: When it comes to orienting users, I feel like "within" is more direct than "of". Making this edit is totally up to you ofc.

```suggestion
You can also chat with the Security Assistant from several areas within {elastic-sec}, and context-specific data and prompts will populate your conversation.
```